cloudify_cli/commands/agents.py

```diff
@@ -79,19 +79,29 @@ def install(deployment_id,
     """
     # install agents across all tenants
     if all_tenants:
+        error_flag = True   # True = has errors
         tenants_list = [tenant.name for tenant in client.tenants.list()]
         for tenant in tenants_list:
```

Comment:
This is becoming a pretty large function. Maybe it would be better to split the all_tenants case and the single tenant case into 2 separate functions.